src/woo_publications/metadata/models.py

```diff
@@ -58,6 +61,43 @@ class InformationCategory(OrderedModel):
         max_length=15,
         default=InformationCategoryOrigins.custom_entry,
     )
+    # Retention fields:
+    bron_bewaartermijn = models.CharField(
+        _("retention source"),
+        help_text=_(
+            "The source of the retention (example: Selectielijst gemeenten 2020)."
+        ),
+        max_length=255,
+        null=True,
+        blank=False,
+    )
+    selectiecategorie = models.CharField(
+        _("selection category"),
+        help_text=_(
+            "The category within the source of the retention (example: 20.1.2)."
+        ),
+        max_length=255,
+        blank=True,
+    )
+    archiefnominatie = models.CharField(
+        _("archive nomination"),
+        help_text=_("Determines if the archived data will be retained or destroyed."),
+        choices=ArchiveNominationChoices.choices,
+        max_length=16,
+        null=True,
+        blank=False,
+    )
+    bewaartermijn = models.PositiveIntegerField(
+        _("retention period"),
+        help_text=_("The retention period in years."),
+        validators=[MaxValueValidator(999)],
+        null=True,
+        blank=False,
```

Comment:
```suggestion
        default=10,
        blank=False,
```
because the field is required, the implication is that values need to be set for existing data. Let's make it clear in the changelog at release time that we've set some defaults that need to be revisited.

Reply:
I changed the default to 10. But do we want to set the defaults ourselves? Without them it's a lot more clear that they still need to be set.

Reply:
Only if you actually open each item in the admin and try to save it again. And fresh instances will get the auto-populated data from fixtures, with empty fields, and then when we try to calculate using this number a lot of stuff will break downstream because no value has been entered, meaning we'll have to deal with those error cases... Ultimately setting a default value here is the only sane approach.

This is always the tricky bit when adding required fields when you have existing data - you need to get a value from _somewhere_ and often that means taking a pragmatic approach.